akd_client/Cargo.toml

```diff
@@ -16,9 +16,10 @@ crate-type = ["cdylib", "rlib"]
 sha2 = { version = "0.10.1", optional = true, default-features = false }
 sha3 = { version = "0.10.0", optional = true, default-features = false }
 blake3 = { version = "1.3.1", optional = true, default-features = false }
+serde_json = "1.0"
```

Comment:
nit: can you loosen this to just ` = "1"` so we get minor upgrade compat (should it occur)?